extensions/interactions/interaction_specs.json

```diff
@@ -997,5 +997,54 @@
       "IsEquivalentTo": "is equivalent to {{f|Fraction}}",
       "HasDenominatorEqualTo": "has denominator equal to {{x|NonnegativeInt}}"
     }
+  },
+  "RatioExpressionInput": {
+    "can_have_solution": true,
+    "show_generic_submit_button": true,
+    "instructions": null,
+    "is_trainable": false,
+    "narrow_instructions": null,
+    "description": "Allows learners to enter ratios.",
+    "needs_summary": false,
+    "name": "Ratio Expression Input",
+    "answer_type": "RatioExpression",
+    "is_linear": false,
+    "display_mode": "inline",
+    "id": "RatioExpressionInput",
+    "customization_arg_specs": [
+      {
+        "description": "Custom placeholder text (optional)",
+        "name": "customPlaceholder",
+        "schema": {
+          "type": "custom",
+          "obj_type": "SubtitledUnicode"
+        },
+        "default_value": {
+          "content_id": null,
+          "unicode_str": ""
+        }
+      },
+      {
+        "description": "Minmium number of terms required",
```

Comment:
Minimum

Reply:
Also, what's the use of this cust arg? Since, even if I set minimum number of terms required to 5, I can still add an answer group like '2:3' with only 2 terms..

Reply:
it would be required for the learners part if i'm not wrong ?

Reply:
If the rule is 2:3 and the minimum terms is 5, how can the learner submit the answer for this?

Reply:
okay got it to let me update that but if I add a check into input validation service for this (would it affect the creator as well ?) i mean i can't check anything for rules/customization arg in editor component